samples/templates/default-azuredeploy.json

```diff
@@ -202,6 +202,7 @@
             "DicomFunctions__DurableTask__ConnectionName": "AzureWebJobsStorage",
             "DicomServer__Features__EnableOhifViewer": "[parameters('deployOhifViewer')]",
             "DicomServer__Features__EnableExtendedQueryTags": "true",
+            "DicomServer__Features__SupportNewBlobFormatForNewService": "true",
```

Comment:
Since templates are generally thought of as idempotent, couldn't someone inadvertently set this on an existing deployment if they're just pulling our file directly?

Reply:
They could enable this flag. My intention is to enable for all the newly created services.